README.md

```diff
@@ -22,15 +22,6 @@ REQUIRED SECTIONS:
 After you're finished please remove all the comments and instructions!
```

Comment:
Great job removing some of the comments from your readme file💪. However, you forgot to remove the comments at the top of the file. In order to keep your code clean, please remove all comments from your code that doesn't directly describe the code itself😎